tsconfig.json

```diff
@@ -7,9 +7,9 @@
     "jsxFactory": "h",
     "jsxFragmentFactory": "Fragment",
     "lib": ["esnext", "dom"],
-    "module": "es6",
+    "module": "esnext",
     "moduleResolution": "node",
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```

Comment:
This is an anti-pattern. We introduced `noImplicitAny: true` from the beginning and it should be kept as is. Of course, this should trigger some errors in your implementation, but let's aim to fix it instead of turning off this flag.